apps/admin-portal/src/components/applications/general-application-settings.tsx

```diff
@@ -237,20 +192,10 @@ export const GeneralApplicationSettings: FunctionComponent<GeneralApplicationSet
             return null;
         }
 
-        if (showRevoke
-            || hasRequiredScopes(featureConfig?.applications, featureConfig?.applications?.scopes?.delete)) {
+        if (hasRequiredScopes(featureConfig?.applications, featureConfig?.applications?.scopes?.delete)) {
 
             return (
                 <DangerZoneGroup sectionHeader="Danger Zone">
-                    { showRevoke && (
-                        <DangerZone
-                            actionTitle="Revoke"
-                            header="Revoke the application"
-                            subheader="This action is reversible but cannot use the same
-                                                    client secret. Please proceed with caution."
-                            onActionClick={ (): void => setShowRevokeConfirmationModal(true) }
-                        />
-                    ) }
                     { hasRequiredScopes(featureConfig?.applications, featureConfig?.applications?.scopes?.delete) && (
```

Comment:
Shall we format the code properly here?

Reply:
what is the formatting issue here?

Reply:
Shouldn't the line with the circuit breaker come to the next line?